storybook/package.json

```diff
@@ -8,23 +8,23 @@
     "storybook-a11y-test": "NODE_PATH=./node_modules node ./scripts/storybook-a11y-test.js"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.5",
+    "@babel/core": "^7.20.12",
     "@shopify/storybook-a11y-test": "^1.1.0",
-    "@storybook/addon-a11y": "^6.5.14",
-    "@storybook/addon-actions": "^6.5.14",
-    "@storybook/addon-backgrounds": "^6.5.14",
-    "@storybook/addon-controls": "^6.5.14",
-    "@storybook/addon-docs": "^6.5.14",
-    "@storybook/addon-links": "^6.5.14",
+    "@storybook/addon-a11y": "^6.5.16",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-backgrounds": "^6.5.16",
+    "@storybook/addon-controls": "^6.5.16",
+    "@storybook/addon-docs": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/addon-viewport": "^6.5.14",
-    "@storybook/addons": "^6.5.14",
-    "@storybook/builder-webpack5": "^6.5.14",
-    "@storybook/manager-webpack5": "^6.5.14",
-    "@storybook/source-loader": "^6.5.14",
-    "@storybook/theming": "^6.5.14",
-    "@storybook/vue3": "^6.5.14",
-    "@vue/compiler-sfc": "^3.2.45",
+    "@storybook/addon-viewport": "^6.5.16",
+    "@storybook/addons": "^6.5.16",
+    "@storybook/builder-webpack5": "^6.5.16",
+    "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/source-loader": "^6.5.16",
+    "@storybook/theming": "^6.5.16",
+    "@storybook/vue3": "^6.5.16",
+    "@vue/compiler-sfc": "^3.2.47",
```

Comment:
Hmmm I think we don't need this anymore do we? Unless maybe storybook doesn't work without it?

https://www.npmjs.com/package/@vue/compiler-sfc

> Note: as of 3.2.13+, this package is included as a dependency of the main vue package and can be accessed as vue/compiler-sfc. This means you no longer need to explicitly install this package and ensure its version match that of vue's. Just use the main vue/compiler-sfc deep import instead.

Reply:
Good catch, it's no longer need to explicitly define here